cmd/run/run.go

```diff
@@ -289,6 +289,15 @@ func run(_ *cobra.Command, _ []string) {
 
 	logger := logger.MustNewLogger(config.Log.Format, config.Log.Level, config.Log.TimestampFormat)
 
+	// Add the startup warning if log-level is set to 'none'
+	logLevel := viper.GetString("log-level")
+	if logLevel == "" {
+		logLevel = os.Getenv("OPENFGA_LOG_LEVEL")
+	}
+	if logLevel == "none" {
+		fmt.Println("WARNING: Logging is not enabled. It is highly recommended to enable logging in production environments to avoid masking attacker operations.")
+	}
+
```

Comment:
Should we move this code inside `config.Verify()`?

```go
if cfg.Log.Level == "none" {
	fmt.Println("WARNING: Logging is not enabled. It is highly recommended to enable logging in production environments to avoid masking attacker operations.")
}
```

So that it's better placed along other code that also does validation on the configs.

Reply:
ahh! I was also confused b/w the root cmd and config file. Will move it under Config verification logic

Reply:
Committed the same in b02221ce49f2deff838ceed287c868f7c7e7a6e3